splinterd/src/node/builder/mod.rs

```diff
@@ -237,6 +239,11 @@ impl NodeBuilder {
         }
     }
 
+    pub fn with_auth_string(mut self, auth: Option<String>) -> Self {
```

Comment:
Perhaps, though this could be named "with_client_auth"

Reply:
Updated